src/components/interaction-designer/ResourceViewer.vue

```diff
@@ -3,7 +3,8 @@
     <slot name="vendor-header" />
     <div class="resource-viewer-contents">
       <resource-viewer-block
-        v-for="block in filteredBlocks"
+        v-for="block in this.activeFlow?.blocks ?? []"
```

Comment:
this can be omitted
(reposting for better visibility)

Reply:
```suggestion
        v-for="block in this.activeFlow?.blocks"
```

Reply:
Sorry about the confusion! I meant `this`
P.S. When do I stop being such a nitpicker?